_data/package_index.yml

```
@@ -608,6 +608,13 @@
   tags: quantum chemistry physics molecular dynamics metadynamics mpi cuda
   license: GNU GPL V2
 
+- name: ABINIT
+  github: abinit/abinit
+  description: ABINIT is a software suite to calculate the optical, mechanical, vibrational, and other observable properties of materials
+  categories: scientific
+  tags: electronic structure calculations quantum chemistry physics molecular dynamics mpi
+  license: GNU GPL V2
```

```suggestion
  version: none
```
No need to specify license here, github has detected it automatically on the repo as GNU GPL v3.
We do need to specify `version: none` since github hasn't detected this automatically.